src/pages/index.tsx

```diff
@@ -241,38 +243,56 @@ const Home: NextPage<HomePageProps> = ({ storefront }) => {
             </div>
           </form>
         </div>
-        <div className="grow">
+        <div className='grow'>
           {nfts.loading ? (
             <>Loading</>
           ) : (
-            <ul className="grid grid-cols-4 gap-6">
+            <div className='grid items-center grid-cols-4 gap-2'>
               {nfts.data?.nfts.map(n => (
-                <li key={n.address}>
-                  <div className="p-4 h-68 overflow-clip hover:bg-gray">
-                    <Link
-                      href={`/nfts/${n.address}`}
-                    >
-                    <a>
-                      <img
-                        src={n.image as string}
-                        alt="nft image"
-                        className="object-fill h-56 pb-2 rounded-lg"
-                      />
-                      <div className="grid grid-cols-2 gap-2">
-                        <div>
-                          <p>{n.name}</p>
+                <div key={n.address} className='h-[450px]'>
+                  <Link href={`/nfts/${n.address}`}>
+                    <div className='h-full p-4 m-0 transition duration-500 transform hover:scale-[1.02]' >
+                      <a>
+                        <div className='w-full border border-b-0 border-[#262626] h-[250px] overflow-clip'>
+                          <img
+                            src={n.image as string}
+                            alt='nft image'
+                            className='w-full h-full pb-2 rounded-lg'
+                          />
                         </div>
-                        <div>
-                          <p className="text-right">55 {solSymbol}</p>
-                          <p className="text-right">Buy Now</p>
+                        <div className='p-4 border border-t-0 border-b-0 border-[#262626]'>
+                          <p className='text-lg'>{n.name}</p>
+                          <p className='text-sm text-[#a8a8a8]'>
+                            <span>
+                              {' '}
+                              <img
+                                src='https://arweave.cache.holaplex.com/jCOsXoir5WC8dcxzM-e53XSOL8mAvO0DetErDLSbMRg'
```

Comment:
storefront logo is available in the database. If not exposed through graphql can you look into that?

Reply:
Yep, will do!